rasa/nlu/extractors/extractor.py

```diff
@@ -119,6 +120,7 @@ def convert_predictions_into_entities(
         tokens: List[Token],
         tags: Dict[Text, List[Text]],
         confidences: Optional[Dict[Text, List[float]]] = None,
+        split_entities_config: Optional[Dict[Text, bool]] = None,
```

Comment:
I guess this should not be optional? Also the doc string for this parameter is missing.